### ISSUE TITLE
Python support

### ISSUE THREAD

**Issue by viktorklang** [cloudstateio/cloudstate#60]
User platform support for Python-based services

**Comment by marcellanz** [comment]
I think Python support would be awesome for a variety of use-cases. If someone hesitates or shy to start with, I have a branch here:
https://github.com/marcellanz/cloudstate_python-support/tree/feature/python-support

that establishes in the most simple way first circles between a User Support Library and the CloudState Proxy and also makes part of the TCK happy.

My Python days are way (way) over, but if someone wants to collaborate or even take over I'm happy if this can help to start serious Python support sooner. Everything starts in `cloud_state.py`

![Screenshot 2019-10-09 at 13 49 46](https://user-images.githubusercontent.com/12616445/66482757-ed1a3f80-eaa3-11e9-96ce-e00af5b672db.png)
(CloudStateEntityDiscovery for the Shopping Cart TCK Example responding to the CloudState Proxy)

![Screenshot 2019-10-09 at 14 02 45](https://user-images.githubusercontent.com/12616445/66482669-bcd2a100-eaa3-11e9-9b7b-e2a992871275.png)
(CloudState TCK with Python having two Tests green)

**Comment by marcellanz** [comment]
While working on CRDT support for the go user library progresses, and sometimes working on multiple issues is fun, I started to get working on python support more seriously.

The repository I started last year with minimal TCK success and a "how to get started":
https://github.com/marcellanz/cloudstate_python-support/tree/feature/python-support

has gotten some interest mainly because of the how to start documentation, but not so much that anyone picked it up. After having also support for languages like Kotlin, Swift and C#, Python is an important language because of its (necessary and preffered) usage in certain fields.

I'd like to see help in getting a idiomatic python API for it which is key for a to be used library. I have opened an issue for that and tagged it with a "help wanted":
https://github.com/marcellanz/cloudstate_python-support/issues/1

So Pythonistas, if you have opinions how python APIs should be crafted or want to join in any other way this is the time :)
